src/modules/flight_mode_manager/tasks/AutoFollowTarget/follow_target_params.c

```diff
@@ -37,22 +37,32 @@
  * Parameters for follow target mode
  *
  * @author Jimmy Johnson <catch22@fastmail.net>
+ * @author Junwoo Hwang <junwoo091400@gmail.com>
  */
 
-/*
- * Follow target parameters
+/**
+ * Dynamic filtering algorithm responsiveness to target movement in Target Estimator
+ *
+ * lower values increase the responsiveness to changing long lat
+ * but also ignore less noise
+ *
+ * @min 0.0
+ * @max 1.0
+ * @decimal 2
+ * @group Follow target
  */
+PARAM_DEFINE_FLOAT(FLW_TGT_RS, 0.1f);
 
 /**
- * Minimum follow target altitude
+ * Follow target height
  *
- * The minimum height in meters relative to home for following a target
+ * The height in meters for following a target
```

Comment:
Yes @unit is included already. And it would be the relative height (depending on which altitude mode we use of course, which would assume different height for the target).

Reply:
you can just specific the unit in the tag. but the various behavior of the parameter should be described

Reply:
Ahaaa you mean the comment that says 'meters', yes that's unnecessary!